eng/pipelines/mono/templates/xplat-job.yml

```diff
@@ -0,0 +1,111 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osSubgroup: ''
+  name: ''
+  helixType: '(unspecified)'
+  container: ''
+  crossrootfsDir: ''
+  stagedBuild: false
+  strategy: ''
+  pool: ''
+
+  # arcade-specific parameters
+  condition: ''
+  continueOnError: false
+  dependsOn: ''
+  displayName: ''
+  timeoutInMinutes: ''
+  enableMicrobuild: ''
+  gatherAssetManifests: false
+
+  variables: {} ## any extra variables to add to the defaults defined below
+
+jobs:
+- template: /eng/common/templates/job/job.yml
+  parameters:
+
+    name: ${{ parameters.name }}
+    displayName: ${{ parameters.displayName }}
+    container: ${{ parameters.container }}
+    condition: ${{ parameters.condition }}
+    dependsOn:
+      - ${{ if ne(parameters.stagedBuild, true) }}:
```

Comment:
I don't think this will ver be true.

Reply:
Right, this is a copy from coreclr. I assumed it will always be true but didn't change it as this would need to be cleaned-up in coreclr as well and I wanted to minimize risk.